tools/util.wdl

```diff
@@ -68,3 +68,44 @@ task prepare_read_groups_for_star {
         read_groups: "The read group portion of a BAM header as a string"
     }
 }
+
+task file_basename {
+    input {
+        File in_file
+        String? suffix
+        Int max_retries = 1
+    }
+
+    command {
+        basename ${in_file} "${suffix}" > stdout.txt
+    }
+
+    runtime {
+        docker: 'stjudecloud/bioinformatics-base:bleeding-edge'
+        maxRetries: max_retries
+    }
+
+    output { 
+        String out = read_string("stdout.txt")
```

Comment:
Potentially,

```suggestion
        String out = read_string(stdout())
```

combined with not writing to file?

Reply:
This was in response to the issue we were discussing with LSF. Jobin suggested this change above and I do think it is worth being explicit about redirecting the output, because it is easy to imagine another backend specification that writes batch system output to the stdout and combines them.